### DISCUSSION TITLE
Is there a way to disable Dynmap-Towny from auto filling land on dynmap?

### DISCUSSION THREAD

**Discussion by Faisl100** [TownyAdvanced/Towny#5402] [Q&A] [answered]
For example, creating a border around some land on dynmap but not claiming the inside, dynmap-towny will show up on dynmap that you claimed the inside even though when you go inside it's just wilderness. Is there a way to disable this?

**Comment by LlmDl** [comment, marked as answer]
There is not, it is a limitation of dynmap-towny. Pl3xmap-Towny and pl3map does not have this issue.
